### ISSUE TITLE
EKS Fargate Support

### ISSUE THREAD

**Issue by cobookman** [pixie-labs/pixie#263]
**Is your feature request related to a problem? Please describe.**
Pixie support the EKS-Fargate runtime.
https://docs.aws.amazon.com/eks/latest/userguide/fargate.html

**Describe the solution you'd like**
Pixie be installable on EKS-Fargate

**Describe alternatives you've considered**
NA

**Additional context**
Fargatge with EKS does not currently allow daemonsets

> Daemonsets are not supported on Fargate. If your application requires a daemon, you should reconfigure that daemon to run as a sidecar container in your pods.

https://docs.aws.amazon.com/eks/latest/userguide/fargate.html

**Comment by isaevdan** [comment]
Would be nice to have!

**Comment by htroisi** [comment]
[Pixie uses eBPF](https://docs.px.dev/about-pixie/pixie-ebpf) to automatically capture telemetry data and so we're blocked on Fargate's current incompatibility with eBPF: https://github.com/aws/containers-roadmap/issues/1027.
